### PR TITLE
Lmaoo228 2

### PR DESCRIPTION
**Pull request by LewisWelch94**

### CODE REVIEW

File: Script/projectController.js [hunk 1]
```diff
@@ -137,15 +104,51 @@ function createTicket()
 {
     var data = new FormData();
     data.append('function', "createTicket");
-    data.append('projectId', document.getElementById("projectId").value);
+    data.append('projectId', document.getElementById("selectedProjectId").innerHTML);
     data.append('reporterKey', document.getElementById("reporterKey").value);
     data.append('summary', document.getElementById("summary").value);
```
Comment (OnikNoor97) [inline Script/projectController.js:109]: Use Jqeury for these getelementbyid

File: Script/projectController.js [hunk 2]
```diff
@@ -137,15 +104,51 @@ function createTicket()
...
     })
+}
+
+function loadTicketsWithProgress(progress) 
+{
+    let selectedProjectId = document.getElementById("selectedProjectId").innerHTML;
+    if (selectedProjectId == 0) return false;
+
+    if (userLevel >= 2) document.getElementById("ticketBtnDiv").innerHTML = 
+        `<button data-toggle="modal" data-target="#projectModal" onclick="createTicketPrompt(${selectedProjectId})">Create Ticket</button>`;
```
Comment (OnikNoor97) [inline Script/projectController.js:126]: Use Jquery

File: Script/server.js [hunk 3]
```diff
@@ -25,16 +25,6 @@ function loadProjectsFromServer()
     })
 }
 
-function loadTicketsFromServer(id)
```
Comment (OnikNoor97) [inline Script/server.js:28]: I believe one of the 'if' uses loadTicketsFromServer() function, remove it

**Review by OnikNoor97 — CHANGES_REQUESTED** [review]

File: Script/server.js (hunk 3 above)
Comment (LewisWelch94) [reply Script/server.js:28]: Fixed

File: Script/projectController.js (hunk 2 above)
Comment (LewisWelch94) [reply Script/projectController.js]: Fixed

File: Script/projectController.js (hunk 1 above)
Comment (LewisWelch94) [reply Script/projectController.js]: Done

**Review by OnikNoor97 — APPROVED** [review]

**Pull request merged**